### ISSUE TITLE
Remember language choice

### ISSUE THREAD

**Issue by deivid-rodriguez** [ruby/www.ruby-lang.org#1827]
Every time I visit ruby-lang.org, I get redirected to https://www.ruby-lang.org/es/, I guess because Spanish is the language of my browser. Since the Spanish translation is always outdated, I have to go to the footer or directly edit the URL to switch to English.

It'd be nice if either my choice would be remembered through a cookie or something, or that the browser language auto-detection was turned off, given that the translations are usually very outdated.

Thanks a lot!

**Comment by olivierlacan** [comment]
The site already has a ton of cookies, I don't see why we couldn't store a language preference as well despite @stomar's objection in https://github.com/ruby/www.ruby-lang.org/pull/1828#issuecomment-427086111. 
![image](https://user-images.githubusercontent.com/65950/48588540-862cee80-e905-11e8-987d-4ab74cd019f0.png)

**Comment by deivid-rodriguez** [comment]
Is there any plan to fix this? I'm happy to try to help myself!

**Comment by ismailarilik** [comment]
I don't agree with the objection. Browsers should open the locale of course. If the language was persistent then changing locale wouldn't mean anything. If you always want the English site then put a bookmark.

**Comment by deivid-rodriguez** [comment]
>  If the language was persistent then changing locale wouldn't mean anything.

What do you mean by this? Of course, the choice should be persistent until you make a different choice in the future. In that case, the next choice should be the new default.

**Comment by deivid-rodriguez** [comment]
This is how multi language sites usually work, since it's user friendly.
